util/devel/test/apptainer/tryit.sh

```diff
@@ -67,7 +67,8 @@ do
       #exit 1
       ;;
     esac
-
+    # Clean up the cloned chapel repositories inside each repo dirs so the VM wont run out of space
+    rm -Rf chapel
```

Comment:
I don't think this is the right place for this change. In particular, `tryit.sh` is meant to support commands that have nothing to do with the Chapel working directory. I can see a few ways to fix it, listed in my order of preference:
 1. My preferred option -- add `provision-scripts/chapel-quickstart-and-cleanup.sh` and the wrapper script `chapel-quickstart-and-cleanup.sh` to run it instead. It would just delete the `chapel` directory at the end of the `&&`ed command sequence.
 2. Request more disk space on the VM so that deleting the `chapel` directories at the end of testing is sufficient

Reply:
DC-OPS increased the disk-space in the VM to 16 GB. Hopefully we should not run into disk space issue.
I removed the code to remove chapel in the latest patch.

If we run into any issues then I will add a wrapper to delete chapel.